cypress/integration/step_definitions/Ui/floatingBathroomVanities.js

```diff
@@ -0,0 +1,35 @@
+import { When, Then, And } from "cypress-cucumber-preprocessor/steps";
+import { homePage } from "../../../pages/homePage";
+import { floatingBathroomVanities } from "../../../pages/floatingBathroomVanities";
+
+let HomePage = null;
+let FloatingBathroomVanities = null;
+
+
+before(() => {
+  HomePage = new homePage();
+  FloatingBathroomVanities = new floatingBathroomVanities();
+});
+
+And("User navigates to Floating Bathroom Vanities Page", () => {
+  HomePage.navigateToFloatingBathroomVanities();
+});
+
+When("User clicks to the Add to Cart button", () => {
+  FloatingBathroomVanities.clickAddToCartButtonWithIndex(21);
```

Comment:
What about parametrize this from feature file? Also, are you sure there will be always 22 products on the page? Maybe choose smaller number like 3 or 4?

Reply:
done